product-service/package.json

```diff
@@ -1,26 +1,35 @@
 {
-  "name": "product-service",
+  "name": "getproductslist",
   "version": "1.0.0",
   "description": "Serverless aws-nodejs-typescript template",
   "main": "serverless.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test:list": "sls invoke local --function getProductsList",
+    "test:id": "sls invoke local --function getProductById --path src/mock/test-id.json",
+    "test:no-id": "sls invoke local --function getProductById --path src/mock/test-bad-id.json",
+    "test": "jest"
```

Comment:
It seems that you have no scripts to deploy your lambdas.

Reply:
It's not difficult to type `sls deploy` )